src/types/Server.ts

```diff
@@ -374,7 +374,9 @@ export enum ServerAction {
   REST_SIGNIN = 'signin',
   REST_SIGNON = 'signon',
   REST_SIGNOUT = 'signout',
-  REST_PASSWORD_RESET = 'passwordreset',
+  REST_PASSWORD = 'password',
+  REST_MAIL = 'mail',
+  REST_EULA = 'eula',
```

Comment:
Well, a mail, password or eula is more close to a data structure than an action :)

An action is more like send (a mail); check (a mail, a password); accept (an eula); reset (a password) ... something you do on a data structure. That's what I mean by putting the action in the URI in ServerAction and let alone the data structure name alone for now ;)